From ff795dd60c5c7b87cae2efd5bc30176b2cf478ae Mon Sep 17 00:00:00 2001
From: Lukasz Majewski <lukma@denx.de>
Date: Wed, 10 Mar 2021 11:58:53 +0100
Subject: [PATCH 4/5] dts: Change pinmux configuration for ENET2_RX_ER pin (to
 be GPIO)

In the new HW setup - the ENET2_RX_ER pin needs to be configured as
GPIO with internal PULL down resistor (to avoid dropping frames).

Linux pinmux driver will not change configuration already set in
U-Boot.

Signed-off-by: Lukasz Majewski <lukma@denx.de>
---
 arch/arm/boot/dts/imx6ull-kie-inverter.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-kie-inverter.dts b/arch/arm/boot/dts/imx6ull-kie-inverter.dts
index 49266c6b93b8..19f212b12a49 100644
--- a/arch/arm/boot/dts/imx6ull-kie-inverter.dts
+++ b/arch/arm/boot/dts/imx6ull-kie-inverter.dts
@@ -263,7 +263,7 @@
 			MX6UL_PAD_GPIO1_IO06__ENET2_MDIO	0x10010
 			MX6UL_PAD_GPIO1_IO07__ENET2_MDC		0x10010
 			MX6UL_PAD_ENET2_RX_EN__ENET2_RX_EN	0x1b0b0
-			MX6UL_PAD_ENET2_RX_ER__ENET2_RX_ER	0x1b0b0
+			MX6UL_PAD_ENET2_RX_ER__GPIO2_IO15       0x80000000
 			MX6UL_PAD_ENET2_RX_DATA0__ENET2_RDATA00	0x1b0b0
 			MX6UL_PAD_ENET2_RX_DATA1__ENET2_RDATA01	0x1b0b0
 			MX6UL_PAD_ENET2_TX_EN__ENET2_TX_EN	0x1b010
-- 
2.20.1

